cgo_flags.go

```diff
@@ -6,4 +6,5 @@ package jemalloc
 // #cgo linux CFLAGS: -DJEMALLOC_PURGE_MADVISE_DONTNEED
 // #cgo darwin CFLAGS: -DJEMALLOC_OSATOMIC -DJEMALLOC_OSSPIN -DJEMALLOC_PURGE_MADVISE_FREE -DJEMALLOC_HAVE_ISSETUGID
 // #cgo darwin CFLAGS: -DJEMALLOC_ZONE -DJEMALLOC_ZONE_VERSION=8
+// #cgo LDFLAGS: -Wl,-undefined -Wl,dynamic_lookup -lm
```

Comment:
not yet. I'm trying to get it basic profiling to work on linux first. Once that's done, I'll worry about making sure it's all neat and clean, including build flags and various DEFINES